qa/runtests.sh

```diff
@@ -31,17 +31,15 @@ function prepare_venv() {
     fi
     pip install -U pip
     python3 "$(which pip3)" install -r requirements.txt
-
+    python3 "$(which pip3)" install -r tests/requirements.txt
 }
 
 check_python_version
 
 [ "$NOVENV" == "1" ] || prepare_venv || exit 1
 
-$(which pip3) install pytest
 $(which pip3) install pytest-cov
```

Comment:
Though it is not relevant to this PR, can you move pytest-cov to `tests/requirements.txt`?